### ISSUE TITLE
paper-button accessibility issues 

### ISSUE THREAD

**Issue by lpalmaro** [PolymerElements/paper-button#7]
Here are the results from the accessibility developer tools audit: 

[Warning] Text elements should have a reasonable contrast ratio (8)See AX_COLOR_01 for more information.[Warning] These elements are focusable but either invisible or obscured by another element (6)See AX_FOCUS_01 for more information.Not applicable tests (9)Passing tests (6)

Please address the contrast issues, as well as the visibility of the focus state, as it is currently quite difficult to see.

**Comment by tjsavage** [comment]
Thanks! Do you know which specific element setup triggered this?

**Comment by lpalmaro** [comment]
Hey,

Unfortunately, I'm not able to copy/paste the full results, only the
summary of what failed and passed. To see the full results, please simply
run the audit.

To run the audit, install the extension, available at
https://chrome.google.com/webstore/detail/accessibility-
developer-t/fpkknkljclfencbdbgkenhalefipecmb?hl=en

Then open Dev Tools, click
on the Audits tab, then run an Accessibility audit.

Thanks!
Laura

**Comment by alice** [comment]
Should provide and document an `alt` attribute similar to `input type=image`.

**Comment by lpalmaro** [comment]
Ping on progress? Thanks!

**Comment by alice** [comment]
Can't see focus styles for flat buttons at all on Yosemite; might be related to https://code.google.com/p/chromium/issues/detail?id=492695

**Comment by cdata** [comment]
These buttons should be bold when focused. I don't think this is great, but we lack sufficient design direction to do anything else.

**Comment by alice** [comment]
There is literally no visible change at all on focus in Yosemite - no text style change, nothing.
